<launch>
	<arg
		name="bv_device_name"
		default="base_valve"/>
	<arg
		name="bv_connection_mode"
		default="serial"/>
	<arg
		name="bv_address"
		default="None"/>
	<arg
		name="bv_port"
		default="COM7"/>
	<arg
		name="bv_simulation"
		default="False"/>
	<node
		name="base_valve"
		pkg="roslabware_drivers"
		type="base_valve"
		args="
			-bvd $(arg bv_device_name)
			-bvc $(arg bv_connection_mode)
			-bvadd $(arg bv_address)
			-bvp $(arg bv_port)
			-bvl $(arg bv_simulation)"
		output="screen"/>
</launch>
